### PR TITLE
Add blog entry about jbossws cxf release plan

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+project and add new things in the upcoming JBossWS 6.3.0 release. In this blog post,
+we will discuss this topic and give an update what will be included in the new release. 
+
+1. JBossWS Galleon Feature Pack
+   [WildFly Galleon](https://docs.wildfly.org/galleon) allows users to create custom distributions of 
+   the WildFly application server. It uses feature pack to specify a set of components to include in 
+   the custom Wildfly server. 
+   In JBossWS 6.3.0, the new jbossws-cxf-feature-pack will be provided. User can install this feature
+   pack in any of released WildFly 27 and WildFly 28 or 28-SNAPSHOT version. This feature pack will be
+   included in JBossWS dist module and created the custom WildFly server to run whole test suite.
```
Comment (rsearls) [inline _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: suggest changing "and created the custom"  to " and creates a custom".

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+2. Cloud Support
+   Containerized java application can be the first step to easily deploy to the cloud and scale up 
+   or down based on demand. In the next JBossWS release, we'll improve JBossWS cloud support and create 
+   examples/docs about how to run a webservice endpoint in docker container. 
+   The Openshift platform provides numerous benefits for running applications like scalability,high 
+   availability,resource efficiency, rollout and rollback. To allow user to easily deploy webservice 
+   application to the Openshift, we'll provide better support and guide to run webservice on OpenShift.
+
```
Comment (rsearls) [inline _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md:27]: suggest changing "and guide to run" to "and guidance to run"

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+3. Clean build
+   In the previous JBossWS releases, we have to use many arguments and maven profiles to run all jbossws
+   tests and build all artifacts. This will be cleaned and improved in the upcoming 6.3.0 release. We'll 
+   deprecate the old maven profiles and make JBossWS can be build with the simple maven build command line
```
Comment (rsearls) [inline _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: suggest changing "and make JBossWS can be build with" to "and make JBossWS buildable with"

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+   `mvn clean install`. 
+
+4. More Jakarta EE support
+   From JBossWS 6.x, we already changed the javax package name to jakarta and imported the new jakarta 
+   dependencies. For the third party dependencies, the bug fix release is also coming, especially CXF 4.0.1
```
Comment (rsearls) [inline _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: suggest replacing 
"For the third party dependencies, the bug fix release is also coming, especially CXF 4.0.1 is one of our interested dependency upgrades.  In the next release, we'll update as many jakarta dependency as we can."
with
"For the third party dependencies, we'll update as many jakarta dependency
as we can, with particular focus on CXF 4.0.1."

**Review by rsearls — CHANGES_REQUESTED** [review]
proposed minor chanes

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md (hunk 1 above)
Comment (jimma) [reply _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: Done.

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md (hunk 3 above)
Comment (jimma) [reply _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: Thanks.Fixed.

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md (hunk 4 above)
Comment (jimma) [reply _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md]: This is much better. Thanks.

File: _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md (hunk 2 above)
Comment (jimma) [reply _posts/2023-04-17-what-is-new-in-jbossws-cxf-630final.md:27]: +1 .Thanks.

**Comment by jimma** [comment]
@rsearls Fixed the things you suggested. Thanks for review.

**Review by asoldano — APPROVED** [review]

**Pull request merged**